### DISCUSSION TITLE
MacOS Ventura "cannot be opened because the developer cannot be verified"

### DISCUSSION THREAD

**Discussion by sumanparia** [helix-editor/helix#6759] [Q&A]
![image](https://user-images.githubusercontent.com/6039696/232018601-0d3a79bb-f21b-4946-b388-950dca0a36e3.png)

This alert is coming for all the so files.
Any quick solution?

**Comment by LEI** [comment]
You might have to right click then "Open" the trusted file from elsewhere first, or [change your security settings](https://stackoverflow.com/a/38238957/7796750), probably not specific to helix.

**Comment by hangj** [comment]
Try this:

```sh
cd ~/.config/helix/runtime/grammars
ls -1 | grep -e '.so$' | xargs xattr -cr 
ls -1 | grep -e '.so$' | xargs codesign --force --deep --sign - 
```

It works for me
